### ISSUE TITLE
PWA icon does not load because manifest specifies relative src paths

### ISSUE THREAD

**Issue by rsesek** [standardnotes/web#499]
Version: 3.5.8

If you install Standard Notes web as a PWA, it loads the manifest from here: https://app.standardnotes.org/favicon/site.webmanifest

That specifies the icons like so:

```json
    "icons": [
        {
            "src": "favicon/android-chrome-192x192.png",
            "sizes": "192x192",
            "type": "image/png"
        },
        {
            "src": "favicon/android-chrome-512x512.png",
            "sizes": "512x512",
            "type": "image/png"
        }
    ],
```

But because the icons are specified using relative paths, and the `site.webmanifest` file is already in the `favicons/` directory, they fail to load.

The following error is observable in the DevTools console:

`Error while trying to use the following icon from the Manifest: https://app.standardnotes.org/favicon/favicon/android-chrome-192x192.png (Download error or resource isn't a valid image)`

The icons’ `src` attribute should either be made into an absolute path, or the `favicon/` prefix should be removed. Currently this causes the PWA to have a generated icon (see attached) rather than the intended app icon:

<img width="194" alt="Screen Shot 2020-11-21 at 4 03 36 PM" src="https://user-images.githubusercontent.com/57350/99887540-30396280-2c13-11eb-973a-8d8f3c2eed0d.png">

**Comment by baptiste-grob** [comment]
Hey @rsesek, I've implemented your suggested fix in v3.5.9. Does it resolve the issue for you? :)

**Comment by rsesek** [comment]
Yup, thanks! The PWA now installs with an icon.
